specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-01-20-preview/guestconfiguration.json

```diff
@@ -555,6 +649,58 @@
         }
       },
       "description": "Report for the guest configuration. Report contains information such as compliance status, reason and more."
+    },
+    "SoftwareList": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Software"
+          },
+          "description": "The list of software installed on a given virtual machine."
+        }
+      },
+      "description": "The list of software installed on a given virtual machine."
+    },
+    "Software": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": false,
+          "$ref": "#/definitions/SoftwareProperties",
+          "description": "Properties of the software."
+        }
+      },
+      "allOf": [
+        {
+        "$ref": "../../common/v1/definitions.json#/definitions/ProxyResource"
+      }
+    ],
+    "description": "Describes a specific software installed on the given virtual machine."
+    },
+    "SoftwareProperties": {
+      "description": "Properties of the software object.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Name of the software"
+        },
+        "version": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Version of the software"
+        },
+        "publisher": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Publisher of the software"
+        },
+        "lastRefreshDate": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Version of the software"
```

Comment:
>Version [](start = 26, length = 7)

last refresh date

Reply:
Thanks, fixed in newer version, removed this from older api version json.